sympy/core/basic.py

```diff
@@ -1896,6 +1896,24 @@ def could_extract_minus_sign(self):
         return False  # see Expr.could_extract_minus_sign
 
 
+class FuncHookMeta(ManagedProperties):
+    def __init__(self, name, bases, dct):
+        pass
+    def __subclasscheck__(cls, subclass):
+        if cls == subclass: return True
```

Comment:
Please make the one-liners `if foo: do` into two lines with indentation on the second and leave a blank line bewteen functions.